client/src/holdings/AddTradeDialog.tsx

```diff
@@ -35,7 +35,7 @@ export default function AddTradeDialog({
   return (
     <div>
       <Button variant='outlined' color='primary' onClick={handleClickOpen}>
-        Add Trade
+        Add Trade ({holding.symbol})
```

Comment:
Ah actually I meant it should go in the dialog title itself rather than the open button. The reason is sometimes you open the dialog and then forget which holding it was actually for! So it's within the element:

`<DialogTitle id='form-dialog-title'>...</DialogTitle>`

Reply:
Need to remove the change from the button as well, as we don't need it there